Makefile

```diff
@@ -780,14 +721,10 @@ get-next-version-increment:
 
 update-resources: SHELL := /bin/bash
 update-resources: check-requirements update-resource-images update-roles update-helmcharts
-	for platform in 'openshift' 'kubernetes'
+	for platform in 'openshift'
```

Comment:
I think we can remove this cycle for if we're supporting only one platform.

Reply:
done